examples/build.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
     implementation project(":basicdataset")
     implementation project(":model-zoo")
     implementation project(":extensions:timeseries")
+    implementation 'org.testng:testng:7.1.0'
```

Comment:
1. Do not use 7.1.0, it has bug
2. testng is already included in testImplemenation
3. testng should not be included as implemention